### PR TITLE
feat: TE5-T104 add javascript using rails practical to the module

### PR DESCRIPTION
**Pull request by chitrak-bhatt**

### CODE REVIEW

File: app/controllers/registerusers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      flash[:errors] = @user.errors.full_messages
+      redirect_to updatepassword_registeruser_path(@registeruser)
+    end
+  end
+
+  private
+  def resgisterusers_params
+    params.require(:registeruser).permit(:first_name,:last_name,:email,:password,:subscription,:subscription_email)
+  end
+end
```
Comment (dhruvika29kevat) [inline app/controllers/registerusers_controller.rb:76]: End of line missing

File: app/views/registerusers/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <br>
+    <%= f.label :subscription_email, "Subscription Email :", class:"mt-3" %>
+    <%= f.email_field :subscription_email ,placeholder: "Enter E-mail for subscription",class:"mb-2 form-control" %>
+    <br>
+    <div class="d-flex">
+    <%= f.submit "Save User", class:"btn-success  mt-4 me-2  border-0 p-2 rounded-pill center" %>
+    <% end %>
+    <%= button_to "View all Users", registerusers_path,method: :get,class:"btn-primary center mt-4 border-0 p-2 rounded-pill" %>
+    </div>
+</div>
```
Comment (dhruvika29kevat) [inline app/views/registerusers/new.html.erb:32]: End of line missing 
add end of line on every file where it is missing.

Comment (chitrak-bhatt) [reply app/views/registerusers/new.html.erb:32]: Changes done, Can you please check.

File: app/controllers/registerusers_controller.rb (hunk 1 above)
Comment (chitrak-bhatt) [reply app/controllers/registerusers_controller.rb:76]: It is present on my IDE.

**Pull request merged**